packages/terra-form-select/src/_Frame.jsx

```diff
@@ -403,12 +485,23 @@ class Frame extends React.Component {
         ref={(select) => { this.select = select; }}
       >
         {/* eslint-disable-next-line jsx-a11y/interactive-supports-focus */}
-        <div role="textbox" aria-disabled={!!disabled} className={cx('display')} onMouseDown={this.openDropdown}>
+        <div
+          role={!disabled ? 'combobox' : undefined}
+          aria-controls={!disabled && this.state.isOpen ? 'terra-select-dropdown' : undefined}
+          aria-disabled={!!disabled}
+          aria-owns={this.state.isOpen ? 'terra-select-dropdown' : undefined}
+          aria-expanded={!!disabled && !!this.state.isOpen}
+          aria-haspopup={!disabled ? 'true' : undefined}
+          aria-label="Search"
+          aria-describedby="screen-reader-text"
+          className={cx('display')}
+        >
+          <span id="screen-reader-text" className={cx('visually-hidden-component')}>
```

Comment:
Should this id be more specific to the select element?

Reply:
Updated in [this commit](https://github.com/cerner/terra-core/pull/2236/commits/77e44ebdb23637b8a9fdc33e3278ba3d0937bb73) to use 
```js
const ariaDescribedById = `terra-select-screen-reader-description-${uniqueid()}`;
```